locales/en/app.ftl

```diff
@@ -334,16 +340,19 @@ user-fb-compromised-single =
   This breach exposed the following personal info. If you haven’t already, 
   change your passwords.
 user-generic-fb-compromised-single = This breach exposed the following personal info.
-guest-fb-compromised-single = 
-  This breach exposed the following personal info. Sign up for a {-brand-fxa} 
-  to get your full report and alerts about new breaches.
 
-guest-fb-compromised-blurb =
+guest-fb-compromised-single-v2 = This breach exposed the following personal info. 
+    Create a free {-brand-fxa} for your full report of past breaches, new breach alerts, 
+    and info about other {-brand-Mozilla} services.
+
+guest-fb-compromised-blurb-v2 =
   { $breachCount ->
-      [one] This email also appeared in { $breachCount } other breach. Sign up for a 
-            {-brand-fxa} to get your full report and alerts about new breaches.
-     *[other] This email also appeared in { $breachCount } other breaches. Sign up for a 
-              {-brand-fxa} to get your full report and alerts about new breaches.
+      [one] This email also appeared in { $breachCount } other breach. Create a 
```

Comment:
I don't know that we have a rule or not, but I did notice in other strings we hardcode "1" or sometimes "one". Not sure it makes a difference to anything in life, but see-something-say-something:

1. [`featured-breach-results[one]`](https://github.com/mozilla/blurts-server/blob/d1e1fcdb9098a89d09bbdbd7f52e4dca201c6fb4/locales/en/app.ftl#L149-L154) = 

    > ```html
    > [one] Your account appeared in the <span class="bold"> { $featuredBreach } </span> breach, as well as one other breach.
    > ```

1. [`featured-breach-not-compromised[one]`](https://github.com/mozilla/blurts-server/blob/d1e1fcdb9098a89d09bbdbd7f52e4dca201c6fb4/locales/en/app.ftl#L156-L161) =

    > ```html
    > [one] Your account did not appear in the <span class="bold">{ $featuredBreach }</span> breach, but did appear in one other breach.
    > ```

1. [`user-scan-results-headline[one]`](https://github.com/mozilla/blurts-server/blob/d1e1fcdb9098a89d09bbdbd7f52e4dca201c6fb4/locales/en/app.ftl#L307-L311) =

    > ```html
    > [one] { $userEmail } appeared in 1 data breach.
    > ```

1. [`guest-scan-results-headline[one]`](https://github.com/mozilla/blurts-server/blob/d1e1fcdb9098a89d09bbdbd7f52e4dca201c6fb4/locales/en/app.ftl#L313-L317) = 

    > ```html
    > [one] This email appeared in 1 data breach.
    > ```

But not always... here's a couple places where we _didn't_ hardcode a number/string for the [one] case:

1. https://github.com/mozilla/blurts-server/blob/d1e1fcdb9098a89d09bbdbd7f52e4dca201c6fb4/locales/en/app.ftl#L163-L168

1. https://github.com/mozilla/blurts-server/blob/d1e1fcdb9098a89d09bbdbd7f52e4dca201c6fb4/locales/en/app.ftl#L327-L331

Reply:
The practical result for en-US is the same, other locales will have to use the variables. Probably good to use the variable at that point.

Reply:
When surfacing the number of breaches a user has, the rule going forward is to use the numeral - unless that numeral is zero. Zero should still get spelled out because it looks too much like an uppercase letter "O". 

I don't think it affects anything but I'll use { $breachCount } where possible instead of hardcoding the 1s in the future.